code/datums/components/crafting/recipes.dm

```diff
@@ -142,6 +142,142 @@
+				/obj/item/assembly/signaler/anomaly/grav = 1,
+				/obj/item/stack/cable_coil = 5,
+				/obj/item/weaponcrafting/gunkit/beam_rifle = 1)
+	time = 200
+	category = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
+
+/datum/crafting_recipe/beam_rifle/New()
+	..()
+	blacklist += subtypesof(/obj/item/gun/energy/e_gun)
+
+/datum/crafting_recipe/ebow
+	name = "Energy Crossbow"
+	tools = list(TOOL_SCREWDRIVER, TOOL_MULTITOOL)
+	result = /obj/item/gun/energy/kinetic_accelerator/crossbow/large
+	reqs = list(/obj/item/gun/energy/kinetic_accelerator = 1,
+				/obj/item/stack/cable_coil = 5,
+				/obj/item/weaponcrafting/gunkit/ebow = 1,
+				/datum/reagent/uranium/radium = 15)
+	time = 200
+	category = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
+
+/datum/crafting_recipe/ebow/New()
+	..()
+	blacklist += subtypesof(/obj/item/gun/energy/kinetic_accelerator)
+
+/datum/crafting_recipe/xraylaser
+	name = "X-ray Laser Gun"
+	tools = list(TOOL_SCREWDRIVER, TOOL_MULTITOOL)
+	result = /obj/item/gun/energy/xray
+	reqs = list(/obj/item/gun/energy/laser = 1,
+				/obj/item/stack/cable_coil = 5,
+				/obj/item/weaponcrafting/gunkit/xray = 1)
+	time = 200
+	category = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
+
+/datum/crafting_recipe/xraylaser/New()
+	..()
+	blacklist += subtypesof(/obj/item/gun/energy/laser)
+
+/datum/crafting_recipe/hellgun
+	name = "Hellfire Laser Gun"
+	tools = list(TOOL_SCREWDRIVER, TOOL_MULTITOOL)
+	result = /obj/item/gun/energy/laser/hellgun
+	reqs = list(/obj/item/gun/energy/laser = 1,
+				/obj/item/stack/cable_coil = 5,
+				/obj/item/weaponcrafting/gunkit/hellgun = 1)
+	time = 200
+	category = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
+
+/datum/crafting_recipe/hellgun/New()
+	..()
+	blacklist += subtypesof(/obj/item/gun/energy/laser)
+
+/datum/crafting_recipe/ioncarbine
+	name = "Ion Carbine"
+	tools = list(TOOL_SCREWDRIVER, TOOL_MULTITOOL)
+	result = /obj/item/gun/energy/ionrifle/carbine
+	reqs = list(/obj/item/gun/energy/laser = 1,
+				/obj/item/stack/cable_coil = 5,
+				/obj/item/weaponcrafting/gunkit/ion = 1)
+	time = 200
+	category = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
+
+/datum/crafting_recipe/ioncarbine/New()
+	..()
+	blacklist += subtypesof(/obj/item/gun/energy/laser)
+
+/datum/crafting_recipe/decloner
+	name = "Biological Demolecularisor"
+	tools = list(TOOL_SCREWDRIVER, TOOL_MULTITOOL)
+	result = /obj/item/gun/energy/decloner
+	reqs = list(/obj/item/gun/energy/laser = 1,
+				/obj/item/stack/cable_coil = 5,
+				/obj/item/weaponcrafting/gunkit/decloner = 1,
+				/datum/reagent/baldium = 30,
+				/datum/reagent/toxin/mutagen = 40)
+	time = 200
+	category = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
+
+/datum/crafting_recipe/decloner/New()
+	..()
+	blacklist += subtypesof(/obj/item/gun/energy/laser)
+
+/datum/crafting_recipe/teslarevolver
+	name = "Tesla Revolver"
+	tools = list(TOOL_SCREWDRIVER, TOOL_MULTITOOL)
+	result = /obj/item/gun/energy/tesla_revolver
+	reqs = list(/obj/item/assembly/signaler/anomaly/flux = 1,
```

Comment:
does this garbage of gun really need an anomaly?

Reply:
Did you read the open post? I said adding anomalies to the recipes gives reason to make these weapons better. They suck right now because they've been hugboxed to shit.

Reply:
but this pr doesnt make them better, and i imagine nobody will care enough to buff it

Reply:
I don't want to do too much in one PR. If I overbloat this one with a bunch of unrelated balance changes, it'll only slow things down. One step at a time.

Reply:
Ill buff them eventually if no one else does